test/e2e/suites/e2e/e2e_test.go

```diff
@@ -161,11 +161,13 @@ var _ = Describe("e2e tests", func() {
 
 			filterNone := ports.ListOpts{}
 			plist1, err := shared.DumpOpenStackPorts(e2eCtx, filterNone)
+
```

Comment:
Adding new line here is out of concern of this PR. Please remove this new line.

Reply:
Done, thanks.